_includes/templates/summaries/linkedin.html

```diff
@@ -0,0 +1,16 @@
+<h2 class="h4">
+    <span class="cf-icon cf-icon-linkedin-square"></span>
+    Follow us on LinkedIn
+</h2>
+<p class="short-desc">
+    The CFPB is one of the most searched for agencies
+    in the federal government.
+    Connect with us to stay updated on the work
+    we do and new opportunities to be a part of it.
+</p>
+<p class="short-desc">
```

Comment:
This isn't a description, should this be something else? Maybe just a link on it's own?